python/main_ssl_disabled.py

```diff
@@ -0,0 +1,31 @@
+import asyncio
+import websockets
+import json
+import ssl
+import pathlib
+from datetime import datetime, timezone
+
+## unsecure ssl approach - TESTING ONLY
+ssl_context = ssl.SSLContext()
+ssl_context.check_hostname = False
+ssl_context.verify_mode = ssl.CERT_NONE
+
+async def connect_ais_stream():
+
+    async with websockets.connect("wss://stream.aisstream.io/v0/stream",ssl=ssl_context) as websocket:
+        subscribe_message = {"APIKey": '6dfe2bbb7da7a37569286a1a2131993083dbb0ce', "BoundingBoxes": [[[-180, -90], [180, 90]]]}
```

Comment:
I will merge this however , could you create another PR without your APIKey ?

Reply:
Woops, that was a silly thing to do! Have removed the key and replaced with placeholder. Have deactivated key also!